client/landing/gutenboarding/components/signup-form/signup-display-anchor.tsx

```diff
@@ -0,0 +1,166 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Button, TextControl, Modal, Notice } from '@wordpress/components';
+import { useI18n } from '@automattic/react-i18n';
+import type { WPElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import ModalSubmitButton from '../modal-submit-button';
+import './style.scss';
+import SignupFormHeader from './header';
+
+interface Props {
+	closeModal: () => void;
+	emailVal: string;
+	errorMessage: string | undefined;
+	handleSignUp: ( event: React.FormEvent< HTMLFormElement > ) => Promise< void >;
+	isFetchingNewUser: boolean | undefined;
+	isMobile: boolean;
+	loginUrl: string;
+	passwordVal: string;
+	recaptcha_tos: WPElement;
+	setEmailVal: React.Dispatch< React.SetStateAction< string > >;
+	setPasswordVal: React.Dispatch< React.SetStateAction< string > >;
+	tos: WPElement;
+}
+
+const SignupDisplayAnchor = ( {
+	closeModal,
+	emailVal,
+	errorMessage,
+	handleSignUp,
+	isFetchingNewUser,
+	isMobile,
+	loginUrl,
+	passwordVal,
+	recaptcha_tos,
+	setEmailVal,
+	setPasswordVal,
+	tos,
+}: Props ) => {
+	const { __ } = useI18n();
+
+	const form = (
+		<form onSubmit={ handleSignUp }>
+			<fieldset className="signup-form__fieldset">
+				<TextControl
+					value={ emailVal }
+					disabled={ isFetchingNewUser }
+					type="email"
+					onChange={ setEmailVal }
+					placeholder={ __( 'Email address' ) }
+					required
+					autoFocus={ ! isMobile } // eslint-disable-line jsx-a11y/no-autofocus
```

Comment:
That's a good question. Mostly, I am taking the current Gutenboarding sign up page and extending it, so that's coming from the current sign-up page here:
https://github.com/Automattic/wp-calypso/blob/cfdb935ebdfb9d3414f45c259499342e2b7caf4c/client/landing/gutenboarding/components/signup-form/index.tsx#L243

I think I traced it back to first being added here: https://github.com/Automattic/wp-calypso/pull/39586 https://github.com/Automattic/wp-calypso/commit/0cc43dad2c8

Reply:
Fair enough. I actually didn't notice it autofocusing when I loaded it anyway, so I'm not super sure this is doing anything and it could probably be taken out. But again, not a blocker.